src/components/CreateCertificate/CreateCertificate.tsx

```diff
@@ -1,25 +1,29 @@
 import { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
 import { useAppSelector } from "store/store";
 
 import Button from "components/Button/Button";
 import Modal from "components/Modal/Modal";
 
 
-import { Address,
-    Value,
-    BigNum,
-    LinearFee,
-    TransactionBuilderConfigBuilder,
-    TransactionUnspentOutputs,
-    TransactionUnspentOutput,
-    TransactionBuilder,
-    TransactionWitnessSet,
-    Transaction,
-    TransactionOutput,
-    CoinSelectionStrategyCIP2
- } from '@emurgo/cardano-serialization-lib-browser';
+// import { Address,
+//     Value,
+//     BigNum,
+//     LinearFee,
+//     TransactionBuilderConfigBuilder,
+//     TransactionUnspentOutputs,
+//     TransactionUnspentOutput,
+//     TransactionBuilder,
+//     TransactionWitnessSet,
+//     Transaction,
+//     TransactionOutput,
+//     CoinSelectionStrategyCIP2
+//  } from '@emurgo/cardano-serialization-lib-browser';
```

Comment:
let's remove this commented code, If in the future is going to be needed we can easily add it